contrib/ubase/Makefile

```diff
@@ -0,0 +1,133 @@
+TOPDIR = $(realpath ../..)
+
+
+HDR = \
+	arg.h \
+	compat.h \
+	crypt.h \
+	fs.h \
+	md5.h \
+	queue.h \
+	sha1.h \
+	sha224.h \
+	sha256.h \
+	sha384.h \
+	sha512.h \
+	sha512-224.h \
+	sha512-256.h \
+	text.h \
+	utf.h \
+	util.h
+
+LIBUTILSRC = \
+	libutil/agetcwd.c        \
+	libutil/agetline.c       \
+	libutil/apathmax.c       \
+	libutil/concat.c         \
+	libutil/ealloc.c         \
+	libutil/eprintf.c        \
+	libutil/estrtol.c        \
+	libutil/estrtoul.c       \
+	libutil/explicit_bzero.c \
+	libutil/passwd.c         \
+	libutil/proc.c           \
+	libutil/putword.c        \
+	libutil/recurse.c        \
+	libutil/strlcat.c        \
+	libutil/strlcpy.c        \
+	libutil/strtonum.c       \
+	libutil/tty.c
+
+BIN = \
+	clear             \
+	getty							\
```

Comment:
Formatting issues - stray backslash sign.

Reply:
Fixed in https://github.com/cahirwpz/mimiker/pull/1371/commits/9d96f1da00f1d03baa86b9a9d4001e679e4e6f00